api/v1alpha1/nodehealthcheck_types.go

```diff
@@ -249,9 +249,8 @@ type UnhealthyNode struct {
 
 	// Remediations tracks the remediations created for this node
 	//
-	//+patchStrategy=merge
-	//+patchMergeKey=type
-	//+optional
+	//+listType=map
+	//+listMapKey=resource
```

Comment:
this won't work (and didn't work before), because the key has to be a scalar type, see `x-kubernetes-list-map-keys
` at https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.27/#jsonschemaprops-v1-apiextensions-k8s-io. Can you please remove the markers please?

Also, why did you remove the `optional` marker?

Reply:
Yes I see that's a good point for the //+listMapKey=resource that should be a scalar type.
The optional marker was removed by mistake by the way :(, just saw it .

Reply:
thanks for the update 👍🏼